streams/src/main/java/org/apache/kafka/streams/state/internals/ChangeLoggingWindowBytesStore.java

```diff
@@ -42,36 +41,35 @@
     ChangeLoggingWindowBytesStore(final WindowStore<Bytes, byte[]> bytesStore,
                                   final boolean retainDuplicates) {
         super(bytesStore);
-        this.bytesStore = bytesStore;
         this.retainDuplicates = retainDuplicates;
     }
 
     @Override
     public byte[] fetch(final Bytes key, final long timestamp) {
-        return bytesStore.fetch(key, timestamp);
+        return wrappedStore().fetch(key, timestamp);
```

Comment:
This is ugly -- we should have access to the wrapped store directly via inheritance. (also below)